.packit.yaml

```diff
@@ -24,8 +24,8 @@ jobs:
       fix-spec-file:
         - "bash .packit.sh"
 
-  - <<: *copr
+    #- <<: *copr
     # Run on commit to main branch
-    trigger: commit
-    branch: main
-    project: podman-next
+    #trigger: commit
+    #branch: main
+    #project: podman-next
```

Comment:
Do you prefer to keep this code commented out for future reference? Or should it be deleted?

Reply:
let's keep this in so it stares us in the face :)